kii_thing_if.c

```diff
@@ -131,6 +131,10 @@ static kii_bool_t prv_init_kii_thing_if(
     kii_thing_if->command_handler.kii_core.http_context.buffer_size =
         command_handler_resource->buffer_size;
 
+        kii_sprintf(kii_thing_if->command_handler.kii_core,
```

Comment:
I think `kii_core` should be `kii_core.sdk_info`. What do you think?

Indent is wrong. Could you fix this?